docs/metrics.md

```diff
@@ -4,30 +4,173 @@ Let's look at the various metrics you can calculate using Valor.
 
-
 ## Classification Metrics
-| Name 	| Description 	|
-|---	|---	|
-| Precision 	| The number of true positives divided by the total number of positive predictions (i.e., the number of true positives plus the number of false positives). 	|
-| Recall 	| The number of true positives divided by the total count of the class of interest (i.e., the number of true positives plus the number of true negatives). 	|
-| F1 	| A weighted average of precision and recall, calculated as `(2 * Precision * Recall)/(Precision + Recall)`. 	|
-| Accuracy 	| The number of true positives divided by the total number of predictions. 	|
-| ROC AUC 	| The area under the Receiver Operating Characteristic (ROC) curve for the predictions generated by a given model. 	|
+
+| Name | Description | Equation |
+|:- | :- | :- |
+| Precision | The number of true positives divided by the total number of positive predictions (i.e., the number of true positives plus the number of false positives). | $$\dfrac{\|TP\|}{\|TP\|+\|FP\|}$$ |
+| Recall | The number of true positives divided by the total count of the class of interest (i.e., the number of true positives plus the number of true negatives). | $$\dfrac{\|TP\|}{\|TP\|+\|FN\|}$$ |
+| F1 | A weighted average of precision and recall. | $$\frac{2 * Precision * Recall}{Precision + Recall}$$ |
+| Accuracy | The number of true predictions divided by the total number of predictions. | $$\dfrac{\|TP\|+\|TN\|}{\|TP\|+\|TN\|+\|FP\|+\|FN\|}$$ |
+| ROC AUC | The area under the Receiver Operating Characteristic (ROC) curve for the predictions generated by a given model. | See [ROCAUC methods](#roc-auc-for-classification). |
 
 ## Object Detection and Instance Segmentation Metrics
 
-| Name 	| Description 	|
-|---	|---	|
-| Average Precision (AP) 	| The weighted mean of precisions achieved at several different recall thresholds for a single Intersection over Union (IOU)*, grouped by class. |
-| AP Averaged Over IOUs 	| The average of several AP metrics, calculated at various IOUs, grouped by class. 	|
-| Mean Average Precision (mAP) 	| The mean of several AP scores, calculated over various classes.	|
-| mAP Averaged Over IOUs 	| The mean of several averaged AP scores, calculated over various classes. 	|
+| Name | Description | Equation |
+| :- | :- | :- |
+| Average Precision (AP) | The weighted mean of precisions achieved at several different recall thresholds for a single Intersection over Union (IoU), grouped by class. | See [AP methods](#average-precision-ap-for-object-detection). |
+| AP Averaged Over IoUs | The average of several AP metrics, calculated at various IoUs, grouped by class. | $$\dfrac{1}{\|thresholds\|} \sum\limits_{iou \in thresholds} AP_{iou}$$ |
+| Mean Average Precision (mAP) 	| The mean of several AP scores, calculated over various classes. | $$\dfrac{1}{\|classes\|} \sum\limits_{c \in classes} AP_{c}$$ |
```

Comment:
classes -> number of classes?

Reply:
the vertical bars signify a quantity

Reply:
i'm familiar with vertical bars as cardinality but can't hurt to define it once before using in this way